### PR TITLE
Chore/dashboard updated

### PR DESCRIPTION
**Pull request by alamuk**
Updated the dashboard libs

### CODE REVIEW

File: apps/home-improvement-hub/src/app/app.tsx [hunk 1]
```diff
@@ -1,44 +1,7 @@
...
-        <AlertTitle>Error</AlertTitle>
-        <AlertDescription>
-          Your session has expired. Please log in again.
-        </AlertDescription>
-      </Alert>
-      <Button variant="outline">Button</Button>
-
-      <FeatureAgents />
+    <div>
+      <h1>Hello App</h1>
```
Comment (oscarlagatta) [inline apps/home-improvement-hub/src/app/app.tsx:4]: Remove the Hello App H1, and return a div with a message welcome to app

File: libs/dashboard/dashboard-ui/src/lib/main-nav.tsx
Comment (oscarlagatta) [inline libs/dashboard/dashboard-ui/src/lib/main-nav.tsx:1]: create a folder for pages and create a folder for components

File: libs/dashboard/dashboard-utility/project.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "dashboard-utility",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/dashboard/dashboard-utility/src",
+  "projectType": "library",
+  "tags": [],
```
Comment (oscarlagatta) [inline libs/dashboard/dashboard-utility/project.json:6]: add the tags definitions, eg. scope:dashboard,type:feature

File: libs/dashboard/features/feature-recent-sales/project.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "feature-recent-sales",
+  "$schema": "../../../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/dashboard/features/feature-recent-sales/src",
+  "projectType": "library",
+  "tags": [],
```
Comment (oscarlagatta) [inline libs/dashboard/features/feature-recent-sales/project.json:6]: add the tags for this library as for the others

**Review by oscarlagatta — CHANGES_REQUESTED** [review]
Please address the comments and call for another review, thank you for your hard work.

**Review by Moshe-Victor — APPROVED** [review]

**Review by oscarlagatta — APPROVED** [review]

**Pull request merged**